src/fmu/ensemble/realization.py

```diff
@@ -866,10 +866,10 @@ def get_eclfiles(self):
             >>> real.find_files("eclipse/model/MYMODELPREDICTION.DATA")
 
         Returns:
-            ecl2df.EclFiles. None if nothing found
```

Comment:
Maybe copy this function for now, and expose both but with a deprecation warning added to `get_eclfiles`. You can then have two import checks: `HAVE_ECL2DF` and `HAVE_RES2DF`.

Reply:
Does this resolve the problem of breaking changes @asnyv ?